### PR TITLE
ROE-1561 adding print button to check answers page

### PR DESCRIPTION
**Pull request by SVarallo**
### JIRA link
https://companieshouse.atlassian.net/browse/ROE-1561

### Change description
Adding print button to check answers page

### Work checklist

- [ ] Tests added where applicable
- [x] UI changes meet accessibility criteria

### Merge instructions

We are committed to keeping commit history clean, consistent and linear. To achieve this, this commit should be structured as follows:

```
<type>[optional scope]: <description>
```

and contain the following structural elements:

- fix: a commit that patches a bug in your codebase (this correlates with PATCH in semantic versioning),
- feat: a commit that introduces a new feature to the codebase (this correlates with MINOR in semantic versioning),
- BREAKING CHANGE: a commit that has a footer `BREAKING CHANGE:` introduces a breaking API change (correlating with MAJOR in semantic versioning). A BREAKING CHANGE can be part of commits of any type,
- types other than `fix:` and `feat:` are allowed, for example `build:`, `chore:`, `ci:`, `docs:`, `style:`, `refactor:`, `perf:`, `test:`, and others,
- footers other than `BREAKING CHANGE: <description>` may be provided.

### CODE REVIEW

**Review by mouhajer-ch — APPROVED** [review]

File: views/check-your-answers.html [hunk 1]
```diff
@@ -59,12 +59,23 @@ <h1 class="govuk-heading-xl">Check your answers before sending your application<
...
-        }) }}
+        <div class="govuk-button-group">
+          {{ govukButton({
+            text: "Continue to payment",
+            attributes: {
+              "id": "submit"
+            }
+          }) }}
+
+          {{ govukButton({
```
Comment (markpit) [inline views/check-your-answers.html:70]: are you able to move the print button govukButton into the views/includes/print-button.html file?
and then have on line 70 {% include "includes/print-button.html" %}
might be worth a try as we can then have all the button code in the includes/print-button

File: views/check-your-answers.html [hunk 2]
```diff
@@ -59,12 +59,23 @@ <h1 class="govuk-heading-xl">Check your answers before sending your application<
...
+            text: "Print this page",
+            classes: "govuk-button--secondary",
```
Comment (markpit) [inline views/check-your-answers.html:72]: should also have a unit test to check that the print button is present, not sure how to check if not present when JS disabled though.

File: test/controllers/check.your.answers.controller.spec.ts [hunk 3]
```diff
@@ -43,7 +43,7 @@ import {
   CHECK_YOUR_ANSWERS_PAGE_TITLE,
   CHECK_YOUR_ANSWERS_PAGE_TRUST_TITLE,
   FOUND_REDIRECT_TO,
-  IDENTITY_CHECKS,
+  IDENTITY_CHECKS, PRINT_BUTTON_TEXT,
```
Comment (markpit) [inline test/controllers/check.your.answers.controller.spec.ts:46]: i'd put the print_button_text on a new line

File: package.json [hunk 4]
```diff
@@ -24,7 +24,7 @@
   "license": "MIT",
   "homepage": "https://github.com/companieshouse/overseas-entities-web#readme",
   "dependencies": {
-    "@companieshouse/api-sdk-node": "^2.0.43",
+    "@companieshouse/api-sdk-node": "^2.0.41",
```
Comment (markpit) [inline package.json:27]: did you mean to revert the api-sdk-node?

Comment (SVarallo) [reply package.json:27]: Probably not. I'll change it back

**Comment by markpit** [comment]
looks ok to me, check with @mouhajer-ch as well

**Pull request merged**